### PR TITLE
LAP-137

### PR DESCRIPTION
**Pull request by Cardona-JE**
Finished the LAP-137, please review

### CODE REVIEW

File: src/main/java/com/litografiaartesplanchas/employeesservice/controller/EmployeeController.java [hunk 1]
```diff
@@ -28,6 +34,28 @@ public ResponseEntity<?> registerEmployee(@RequestBody Employee employee) {
...
+
+    @GetMapping("/{id}")
+    public ResponseEntity<?> getEmployeeById(@PathVariable Integer id) {
+        Optional<Employee> employeeOptional = employeeService.getClientById(id);
+        if (employeeOptional.isEmpty()) {
+            return ResponseEntity.status(HttpStatus.NOT_FOUND).body("{\"status\": 404, \"message\": \"Not Found\"}");
+        }
+        Employee employee = employeeOptional.get();
+        return ResponseEntity.ok(employee);
+    }
```
Comment (Juansecod) [inline src/main/java/com/litografiaartesplanchas/employeesservice/controller/EmployeeController.java]: If thows a Exception?

File: src/main/java/com/litografiaartesplanchas/employeesservice/service/EmployeeService.java [hunk 2]
```diff
@@ -15,6 +18,14 @@ public EmployeeService(EmployeeRepository employeeRepository) {
         this.employeeRepository = employeeRepository;
     }
 
+    public List<Employee> getAll(){
+        return employeeRepository.findAll();
+    }
```
Comment (Juansecod) [inline src/main/java/com/litografiaartesplanchas/employeesservice/service/EmployeeService.java]: Error Management?

File: src/main/java/com/litografiaartesplanchas/employeesservice/service/EmployeeService.java [hunk 3]
```diff
@@ -15,6 +18,14 @@ public EmployeeService(EmployeeRepository employeeRepository) {
...
+
+    public Optional<Employee> getClientById(Integer id) {
+		return employeeRepository.findById(id);
+    }
```
Comment (Juansecod) [inline src/main/java/com/litografiaartesplanchas/employeesservice/service/EmployeeService.java]: Error Management?

**Review by Juansecod — CHANGES_REQUESTED** [review]

**Comment by Cardona-JE** [comment]
Errors resolved, please review

**Review by Juansecod — APPROVED** [review]

**Pull request merged**